mapstory/static/style/site/search.less

```diff
@@ -1,243 +1,185 @@
-/* --- Search --- */
-.panel-group{
-    margin: 0;
-    padding: 0;
-}
-
-.panel-default {
-    border: none;
-    background-color: @white;
-    box-shadow: 0;
-}
-
-.panel-default  .panel-heading {
-    border: 1px solid transparent;
+.explore-tabs {
+    li {
+        float: left;
+        list-style: none;
+        font-size: 0.9em;
+        margin: 10px 40px 10px 0;
+        padding: 5px 15px;
+        color: @gray55;
+        font-weight: 400;
+    }
+    .active {
+        font-weight: 600;
+        border-bottom: 2px solid @primary;
+        color: @gray30;
+    }
 }
 
-.panel-title {
-    border: 1px solid transparent;
-    background-color: @white;
-    font-size: 1.2em;
-    color: @gray30;
+.explore-filters {
+    clear: both;
+    h2 {
+        font-size: 1em;
+    }
 }
 
-.panel-body {
-    border: 1px solid transparent;
-    background-color: @white;
+.showing-results, .sort-by {
+    font-size: 0.85em;
+    font-weight: 400;
 }
 
-.search-bar {
-    padding: 20px 0;
+.showing-results {
+    float: left;
+    margin-bottom: 16px;
 }
 
-.whats-hot {
-    ul {
-        list-style-type: none;
-    }
+.sort-by {
+    float: right;
 }
 
-.search-results {
```

Comment:
I checked the code base to see where these classes (which I've written new styles for) are used. The only places after my changes that still had a few of the `.search-results` classes are templates that as far as I can tell never get rendered and aren't actually used on MapStory (like `mapstory/mapstory/templates/groups/initiative_members.html`, `mapstory/mapstory/templates/search/_result_content_min.html`, `mapstory/mapstory/templates/groups/organization_visit.html`, etc.). I'd appreciate someone double-checking me here, though.